app/student/html/html_grading.php

```diff
@@ -1 +1,138 @@
-<p>This is the html grading page. Put your content here.</p>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script>
+$(document).ready(function(){
+    $("#aparna").click(function(){
+        $("#foo").toggle();
+    });
+});
+</script>
+<style>
+body {
+    font-family: "Lato", sans-serif;
+}
+
+td, th {
+    border: 1px solid #dddddd;
+    text-align: left;
+    padding: 8px;
+}
+
+
+.sidenav {
+    height: 100%;
+    width: 250px;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    background-color:  #023F80;
+    overflow-x: hidden;
+    padding-top: 20px;
+}
+
+.sidenav a {
+    padding: 6px 6px 6px 32px;
+    text-decoration: none;
+    font-size: 25px;
+    color: #818181;
+    display: block;
+}
+
+
+#first {
+  background-color:white;
+  width:300px;
+  float:left;
```

Comment:
this is causing your content to be written on top of the portal's footer. It doesn't seem necessary